src/index/Index.h

```diff
@@ -557,9 +558,23 @@ class Index {
       TripleVec& data, const vector<size_t>& actualLinesPerPartial,
       size_t linesPerPartial);
 
-  size_t passContextFileForVocabulary(const string& contextFile);
-
-  void passContextFileIntoVector(const string& contextFile, TextVec& vec);
+  // Generator that returns all words in all the IRIs and literals.
+  //
+  // TODO: Use this generator for both cases (reading the text records from a
+  // file and taking the IRIs and literals as text records). The rename is to
+  // `wordInTextRecords`.
+  //
+  // TODO: So far, this is limited to the internal vocabulary (still in the
+  // testing phase, once it works, it should be easy to include the IRIs and
+  // literals from the external vocabulary as well).
```

Comment:
I think your current include only literals. Do we even want to include IRIs?

Reply:
I think it depends on the data. For example, in DBLP you have IRIs like `<https://dblp.org/rec/books/sp/Behnke03>`. Words like `https` or `dblp` or `org` are probably not so relevant here, but words like `books` or `behnke` might be. Maybe @mra42 can comment.

Reply:
@mra42 Good to know, thanks! QLever dblp already has full-text search on literals enabled, for example: https://qlever.cs.uni-freiburg.de/dblp/f8aZYc . You can enter multiple keywords in the string and you can add a star. The current minimum prefix length is four, but that is easy to change. Is that enough for your needs or is there anything missing?